src/main/java/graphql/execution/batched/BatchedExecutionStrategy.java

```diff
@@ -201,6 +202,52 @@ private void executeImpl(ExecutionContext executionContext,
 
     }
 
```

Comment:
Mostly moved this up in the file so its near where it gets called and tweaked the DF fetching to use the Async code